src/Client/BitlyClient.php

```diff
@@ -49,9 +49,9 @@ public function __construct(ClientInterface $client, $token)
      * @throws \Shivella\Bitly\Exceptions\AccessDeniedException
      * @throws \Shivella\Bitly\Exceptions\InvalidResponseException
      */
-    public function getUrl(string $url, ?string $domain = null, ?string $group_guid = null): string
+    public function getUrl(string $url, ?string $domain = null, ?string $group_guid = null, ?string $title = null, $requestUrl = 'https://api-ssl.bitly.com/v4/shorten'): string
     {
-        $requestUrl = 'https://api-ssl.bitly.com/v4/shorten';
+        ## $requestUrl = 'https://api-ssl.bitly.com/v4/shorten';
```

Comment:
Just wondering, but why is the requestUrl now a function param?

Reply:
To use it in the new method create():
` public function create(string $url, ?string $title = null, ?string $domain = null, ?string $group_guid = null){
        return $this->getUrl($url, $domain, $group_guid , $title, 'https://api-ssl.bitly.com/v4/bitlinks');
    }`